cdap-docs/admin-manual/source/operations/cdap-ui.rst

```diff
@@ -38,5 +38,234 @@ and are confident of their suitability and compatibility. Earlier versions (and
 suppliers) of components may work, but have not necessarily been either tested or
 confirmed compatible.
 
+UI Customization
+---------------------------------
+Certain properties in the UI can be customized. The ``ui/server/config/themes/default.json`` file contains these properties, along with their default values. As of version 1.0, here is the list of properties that can be customized, and their expected values:
```

Comment:
It is also unclear to a user where this UI directory is. Is there a place in the documentation that describes where the UI is installed? Would be good to check on a cluster - perhaps something like `<CDAP_HOME>/ui/...`

Reply:
Yeah it's `<CDAP_HOME>/ui/...`